base/applications/osk/main.c

```diff
@@ -585,12 +585,35 @@ int WINAPI wWinMain(HINSTANCE hInstance,
                     int show)
 {
     HANDLE hMutex;
+    DWORD dwError;
     INT LayoutResource;
 
     UNREFERENCED_PARAMETER(prev);
     UNREFERENCED_PARAMETER(cmdline);
     UNREFERENCED_PARAMETER(show);
 
+    /*
+        Obtain a mutex for the program. This will ensure that
+        the program is launched only once.
+    */
+    hMutex = CreateMutexW(NULL, FALSE, L"OSKRunning");
```

Comment:
Any particular reason to rename the mutex?

Reply:
That's the name used by Windows for the single-instance OSK.
So that, if you test ReactOS' OSK on Windows, while Windows' OSK is open, you don't get the two OSKes opened in parallel.

Reply:
Ah, I checked on Win10 and didn't see any mutex heh. Right, I see OSKRunning on 2003.

Reply:
ProcessExplorer must be run in admin mode, and the mutant name should be displayed.